test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -135,6 +141,23 @@ connection_simulator::query_timestamp(
     uint64_t ts;
     if (query_timestamp == "all_durable") {
         ts = _durable_ts;
+
+        for (auto &session : _session_list) {
+            if (session->is_txn_running()) {
+                uint64_t durable_ts;
+                if (session->is_durable_ts_set())
+                    durable_ts = session->get_durable_timestamp();
+                else if (session->is_commit_ts_set()) {
+                    if (session->is_txn_prepared())
+                        continue;
+                    durable_ts = session->get_first_commit_timestamp();
+                } else
+                    continue;;
```

Comment:
Typo: two `;`. But this should be removed if you decide to go along with my previous comment.

Reply:
done in [88c2d1c](https://github.com/wiredtiger/wiredtiger/pull/8398/commits/88c2d1c196ccd2e39dce10e4fdbc8d552469d4c6)